Examples/test-suite/li_carrays.i

```diff
@@ -2,6 +2,11 @@
 
 %warnfilter(SWIGWARN_RUBY_WRONG_NAME) doubleArray; /* Ruby, wrong class name */
 
+#ifdef SWIGFORTRAN
+%fortran_bindc_struct(AB);
+%fortran_bindc_struct(XY);
```

Comment:
If Fortran can't support carrays.i out the box, then the additional use of %fortran_bindc_struct work needs to be documented.

Reply:
It does. I've moved the fortran-specific testing of array-bound structs to the `fortran_bindc` test.